packages/rx_bloc_cli/doc/Fastlane.md

```diff
@@ -0,0 +1,82 @@
+### Fastlane
```

Comment:
Please ensure there is a link from the main readme.md file to the Fastlane.md file